modules/examiner/php/examiner.class.inc

```diff
@@ -31,13 +31,13 @@ class Examiner extends \NDB_Menu_Filter_Form
     /**
      * Checks if the user has access to examiner menu filter
      *
-     * @return boolean
+     * @param \User $user The user whose access is being checked
+     *
+     * @return bool
      */
-    function _hasAccess()
+    function _hasAccess(\User $user) : bool
     {
-        $user = \User::singleton();
-        return $user->hasPermission('examiner_view')
-            || $user->hasPermission('examiner_multisite');
+        return $user->hasPermission('examiner_view');
```

Comment:
should removing the 'examiner_multisite' permission be done in this PR?

Reply:
when rebasing it it looked like that's what it was on major in how the conflicts were displayed, I must have misread it. I'll add it back.